src/lib/contact-generator/contact-generator.ts

```diff
@@ -84,7 +84,7 @@ export class ContactGenerator {
   }
 
   private contactFrom(item: License | Transaction, info: ContactInfo | PartnerBillingInfo): GeneratedContact {
-    const [firstNameRaw, ...lastNameGroup] = (info.name || ' ').split(' ');
+    const [firstNameRaw, ...lastNameGroup] = (info.name || ' ').replace(/[<>]/, '').split(' ');
```

Comment:
Is it expected to have no more than 1 match? Otherwise replace all should be used `/[<>]/g`